ssas/service/server.go

```diff
@@ -325,6 +335,46 @@ func (s *Server) VerifyClientSignedToken(tokenString string, trackingId string)
 	}
 	return jwt.ParseWithClaims(tokenString, &CommonClaims{}, keyFunc)
 }
+
+func (s *Server) GetSystemIDFromMacaroon(issuer string) (string, error) {
```

Comment:
Do we want a single-purpose method for finding a particular 1st party caveat? Won't the code be 90% the same for finding the group ID from the macaroon?

Reply:
done